src/pair_potential_types.F

```diff
@@ -1795,6 +1830,85 @@ SUBROUTINE pair_potential_nequip_release(nequip)
       END IF
    END SUBROUTINE pair_potential_nequip_release
 
+! **************************************************************************************************
+!> \brief Creates the ALLEGRO potential type
+!> \param allegro ...
+!> \author Teodoro Laino [teo] 11.2005
```

Comment:
Replace it with your name.

Reply:
Done both for NequIP and Allegro, thanks. In general, should one claim authorship of a routine even if it was only slightly modified from a previous one, as it was the case, for example, for all the create, copy, release, clean routines in ```src/pair_potential_types.F``` for each manybody or pair potential?

Reply:
Because we have the git logs, there is not necessarily the need to add the authorship but if you do, just add a hint that it was copied from code by Teodoro.

Reply:
> In general, should one claim authorship of a routine even if it was only slightly modified from a previous one,...

It depends ;-)

For simple boilerplate code like this, you can safely put your own name to let others know that you're the Allegro-expert. For more complex code you can share the credit via the `\history` tag.

Generally, we try to give [more visibility to recent work](https://github.com/cp2k/cp2k/blob/b234498075e47956c66564e93abbe409d5e88a7d/src/common/reference_manager.F#L230-L231).